제이_지유진/3주차_미션/week3/src/components/CreditCard.tsx

```diff
@@ -0,0 +1,25 @@
+import type { TCredit } from "../types/credit";
+
+function CreditCard(credit: TCredit) {
+  return (
+    <div className="flex flex-col items-center justify-between">
+      <img
+        className="rounded-full h-[80px] w-[80px] object-cover border-white border-2"
+        src={
+          credit.profile_path
```

Comment:
이렇게 작성하는 대신
```suggestion
           const URL = credit.profile_path
        ? `https://image.tmdb.org/t/p/w500${credit.profile_path}`
        : 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSU6qxvaanw0gI1HMNRdtIbIBQ9L9STrF2G-w&s';
```
상단에 이렇게 적고 src에 URL만 넘겨주면 가독성이 더 좋을 것 같아요!